pages/dkp/kommander/2.2/networking/use-istio/index.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: layout.pug
+navigationTitle: Use Istio
+title: Use Istio
+menuWeight: 24
+excerpt: Learn how to integrate microservices managed by Istio into a DKP cluster
+beta: false
+experimental: true
+---
+
+
+Istio helps you manage cloud-based deployments by providing an open-source service mesh to connect, secure, control, and observe microservices.
+
+This topic describes how to expose an application running on the DKP cluster using the LoadBalancer (layer-4) service type.
+
+<p class="message--note"><strong>NOTE: </strong>Usage and installation of Istio on DKP is currently a self-service feature.</p>
+
+## Before you begin
+Before you begin, verify the following:
+
+- You must have access to a Linux, macOS, or Windows computer with a supported operating system version.
+
+- You must have a properly deployed and running cluster.
+
+- You must have `cert-manager` enabled and deployed through your `cluster.yaml` configuration.
+
+## Deploy Istio using DKP
+
+Review the [list of available applications](../../workspaces/applications/platform-applications#workspace-platform-applications) that can be enabled to deploy to your attached cluster.
+
+1.  Enable a supported application to deploy to [your existing attached cluster](../../clusters/attach-cluster/) with an `AppDeployment` resource.
```

Comment:
Instead of the generic 'a supported application', since this is a page about installing istio, we should mention that here.

```suggestion
1.  Enable the deployment of Istio to [your existing attached cluster](../../clusters/attach-cluster/) by creating an `AppDeployment` resource.
```